minifi/minifi-nar-bundles/minifi-framework-bundle/minifi-framework/minifi-runtime/pom.xml

```diff
@@ -76,5 +77,11 @@ limitations under the License.
             <groupId>com.fasterxml.jackson.core</groupId>
             <artifactId>jackson-databind</artifactId>
         </dependency>
+
+        <dependency>
+            <groupId>ch.qos.logback</groupId>
+            <artifactId>logback-classic</artifactId>
+            <scope>test</scope>
+        </dependency>
```

Comment:
Logback should not be used as a test dependency. If logging is required in tests, `slf4j-simple` prints to the console. Is there any reason to include Logback in this module for testing?